documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +51,34 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+  labels:
+    createdForProviderType: <provider_type> <2>
 type: Opaque
 stringData:
-  user: <user> <1>
-  password: <password> <2>
-  cacert: | <3>
-    <engine_ca_certificate>
-  thumbprint: <vcenter_fingerprint> <4>
-EOF
+  user: <user> <3>
+  password: <password> <4>
+  domainName: <domain_name> <5>
+  projectName: <project_name> <6>
+  regionName: <region name> <7>
+  cacert: | <8>
+    <ca_certificate>
+  url: <api_end_point> <9>
+  thumbprint: <vcenter_fingerprint> <10>
 ----
-<1> Specify the vCenter user or the {rhv-short} {manager} user.
-<2> Specify the user password.
-<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
-<4> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> The `ownerReferences` section of the manifest is for {osp} only.
```

Comment:
@RichardHoch as @bennyz reacted with +1 on the previous comment on this, let's drop this bullet please (so it won't appear as OpenStack-only)